libfwupdplugin/fu-device.h

```diff
@@ -574,6 +574,16 @@ typedef guint64 FuDeviceInternalFlags;
  */
 #define FU_DEVICE_INTERNAL_FLAG_UPDATE_PENDING (1ull << 35)
 
+/**
+ * FU_DEVICE_INTERNAL_FLAG_ENFORCE_REQUIRES:
+ *
+ * The device uses a generic instance ID and firmware requires a parent, child, sibling or CHID
+ * requirement.
+ *
+ * Since: 1.9.8
+ */
+#define FU_DEVICE_INTERNAL_FLAG_ENFORCE_REQUIRES (1llu << 36)
```

Comment:
nit: 1ull ?

Reply:
Ohh, indeed. I'm not 100% which is the correct way around -- I think we use both in the codebase!

Reply:
**Both are allowed by the C standard** --  (section 6.4.4.1)